### DISCUSSION TITLE
How to delete a Datapane report

### DISCUSSION THREAD

**Discussion by sh-mukherjee** [datapane/datapane#37] [Help] [answered]
I made a couple of example reports on Datapane (free plan) and now I want to delete them. I can't see any way to do so on the Datapane website, so I wonder if somebody can teach me how?

**Comment by lanthias** [comment, marked as answer]
Thanks for reaching out! You can delete a report through the Datapane web UI.

If you're logged in and you are viewing your own report, you should see a delete button on the top right next to the version information:

![image](https://user-images.githubusercontent.com/3541695/98706047-46733300-2376-11eb-9a2e-96764eddcd0a.png)

Please let me know if this is not the case and I will dig into it.

Thanks!

Leo

**Reply by mands** [reply]
Just to add to this, you can also delete a report from the CLI by running `datapane report delete {NAME}`

```bash
$ datapane report delete --help
Usage: datapane report delete [OPTIONS] NAME

  Delete a report

Options:
  --help  Show this message and exit.
```
